Finish Sicko Mode article

mluogh opened the pull request:
Please read for typos and quality.

mluogh committed:
add Sicko Mode article

RahulTirumalareddy commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,39 @@
+
+A less careful listener may simply dismiss Travis Scott and Aubrey “Drake” Graham's hit song *Sicko Mode* as a mere modern pop trap record, perhaps noting the song’s strange choice of three different beats before requesting a change of song to Shack Wes' *Mo Bamba*. The risk of such negligent listening is to miss the song’s true message: the story of a young man exploring his sexuality. 
+
+The first beat of the song is mild and unassuming - a careful allegory chosen to represent our society’s heteronormativity. When our protagonist is in the stage this song describes (i.e. has still not come out), by obeying the heterosexual norms of society, he is repressed just like the beat of the song. Aubrey “Drake” Graham (hereby referred to as solely as Drake), plays a close friend of our pr ...
```
"hereby referred to as solely as Drake" - one too many 'as's, perhaps replace with "hereby referred to solely as Drake"

mluogh replied:
Good catch.

RahulTirumalareddy commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,39 @@
+
+A less careful listener may simply dismiss Travis Scott and Aubrey “Drake” Graham's hit song *Sicko Mode* as a mere modern pop trap record, perhaps noting the song’s strange choice of three different beats before requesting a change of song to Shack Wes' *Mo Bamba*. The risk of such negligent listening is to miss the song’s true message: the story of a young man exploring his sexuality. 
+
+The first beat of the song is mild and unassuming - a careful allegory chosen to represent our society’s heteronormativity. When our protagonist is in the stage this song describes (i.e. has still not come out), by obeying the heterosexual norms of society, he is repressed just like the beat of the song. Aubrey “Drake” Graham (hereby referred to as solely as Drake), plays a close friend of our pr ...
```
"Unable to see our hero live a lie of a life, our artists take action by awakening his true sexuality..." - the second half of this sentence is a bit confusing, as there are a lot of pronouns. I'd suggest replacing 'our artists' and 'his' with a more explicit description.

mluogh replied:
I don't want to replace 'our artists'. One of the jokes here is that I take time to say ([Aubrey Graham] is hereby solely referred to as Drake) but I never explicitly mention him again.

RahulTirumalareddy commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,39 @@
+
+A less careful listener may simply dismiss Travis Scott and Aubrey “Drake” Graham's hit song *Sicko Mode* as a mere modern pop trap record, perhaps noting the song’s strange choice of three different beats before requesting a change of song to Shack Wes' *Mo Bamba*. The risk of such negligent listening is to miss the song’s true message: the story of a young man exploring his sexuality. 
+
+The first beat of the song is mild and unassuming - a careful allegory chosen to represent our society’s heteronormativity. When our protagonist is in the stage this song describes (i.e. has still not come out), by obeying the heterosexual norms of society, he is repressed just like the beat of the song. Aubrey “Drake” Graham (hereby referred to as solely as Drake), plays a close friend of our pr ...
```
We should probably capitalize the 'i's in quotes.

mluogh replied:
I agree, good catch.

RahulTirumalareddy commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,39 @@
+
+Here, as a woman attempts to engage in promiscuous behaviour, our hero denies her access to his room (and therefore himself). Furthermore, he corrects her - his desire for women cannot be compared to the vast ocean, but just a small pool, by comparison practically non-existent.
+
+However, due to the anti-homosexual sentiments still present in many American communities, our protagonist is unable to sustain his true desires. Once they find out, he is physically assaulted and knocked “out like a light **(ey-uh)**” (emphasis mine). The beat change here again represents another change in his life. He must settle for practising his sexuality in secret and pretending to obey het ...
```
not sure what you mean by the '(emphasis mine)' part

mluogh replied:
It is a joke.

mingyli replied:
It is a display of humour.

RahulTirumalareddy reviewed (changes requested):
Hey Michael,
Great piece overall, and I really enjoyed reading it! Please address a few comments regarding semantics and formatting.
I'd also add a hyperlink to the song or an Apple Music/Spotify widget so that the reader has easy access to the piece being discussed. See Ming's post for an example of the former, and my own for the latter.

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,39 @@
+title:  "\"Sicko\" Mode: a modern critique of homophobia in America"
+author: Michael Luo
+date:   2019-04-16
+categories: drake travis scott michael
```
I think your name should not be a topic

mluogh replied:
I think eventually we may wish to allow users to filter by their favourite author.

mingyli replied:
That is available through the field `author`

mluogh replied:
Perhaps.

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,39 @@
+
+The second beat is more aggressive and powerful, consistent with our character’s newfound interest in masculinity. 
+
+> “Different coloured chains, think my jeweller really sellin’ fruits”
```
Remove double quotes in quotations

mluogh replied:
Good catch.

mingyli reviewed (changes requested):
Great work. 

- Please rebase on top of master
- Please make text wrap at textwidth=80. You can accomplish this in vim by using the command gq
- When you write em dashes, you should use two hyphens: --

mluogh commented:
I have addressed your concerns.

1) various typo changes
2) various formatting and code quality changes
3) rebased on master
4) added included astroworld art cover and apple music embeds

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+author: Michael Luo
+date:   2019-04-16
+categories: drake travis scott
+album_art: assets/images/astroworld.png
```
Let's align these attributes = )

mluogh replied:
Good catch.

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+
+> Different coloured chains, think my jeweller really sellin’ fruits
+
+ He immediately affirms his new sexuality by buying multicoloured jewellery,
```
Is this spacing intentional?

mingyli replied:
Also jewellery is spelled wrong

mluogh replied:
Jewellery is British English. Valid.

RahulTirumalareddy replied:
Please address the extraneous spacing at the beginning of the line.

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+The second beat is more aggressive and powerful, consistent with our
+character’s newfound interest in masculinity.
+
+> Different coloured chains, think my jeweller really sellin’ fruits
```
spelling

mluogh replied:
British English.

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+life. He must settle for practising his sexuality in secret and pretending to
+obey heterosexual norms.
+
+He regrets this as he notes “this shit got double bedroom”, meaning he
```
Put comma inside quotes

mluogh replied:
Again, valid otherwise.

mingyli replied:
It's not valid. See my previous link and screenshot

RahulTirumalareddy replied:
Agree with Ming on this one; since the quotes are used to describe speech, commas go inside the marks.

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+obey heterosexual norms.
+
+He regrets this as he notes “this shit got double bedroom”, meaning he
+unfortunately has to physically separate each of his sexualities - one bedroom
```
em dash

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+
+Unfortunately, as the song title suggests, he is eventually unable to
+conclusively assert his preferences. He remains in fear of being branded a
+“sicko”, and this song is a sad reminder of that time his peers had thought he
```
swap comma and quote

mluogh replied:
https://english.stackexchange.com/questions/114375/how-to-use-quotes-in-a-list-with-commas

It is valid otherwise. Furthermore, when quoting in order to distinguish the word, I believe it does not make sense to have the comma inside. We are trying to stress the word "sicko" not the word "sicko,".

mingyli replied:
That is a common misconception. Please see my attached link. You are quoting an excerpt, and as such your citation does not apply

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+
+For those further interested, my colleagues DJ Splash Drip and Kurosare have
+taken this analysis further in their song “[Dicko
+Mode](https://www.youtube.com/watch?v=ycd6Xchkq-4)”.
```
either swap period and quote or remove quotes altogether

mluogh replied:
It does not make sense to have to the period inside. It is not part of the song title.

mingyli replied:
Sire, I refer you to https://www.grammarly.com/blog/quotation-marks/

![image](https://user-images.githubusercontent.com/5828246/56323277-2dfe5080-6120-11e9-9ba6-f18c62eeeb87.png)

mluogh replied:
Sire, I refer _you_ to the Economist and Guardian style guide.

https://english.stackexchange.com/questions/23/is-it-ever-acceptable-for-a-period-to-come-after-a-quote-at-the-end-of-a-sentenc

RahulTirumalareddy replied:
Gentlemen, you both are correct. In American English, periods and commas go inside the quotation mark with no exceptions. However, in British English, the same thing applies for quotes being used for dialogue, but when used to refer to titles or specific terms, the quoted portion is treated as an atomic unit, with punctuation going outside.
Personally, I write in the style of the latter, and favor the British style, as the American style is an anarchic convention. Please refer to https://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks:

> Compositors―people who layout printed material with type―made the original rule that placed periods and commas inside quotation marks to protect the small metal pieces of type from breaking off the end of the sentence. The quotation marks protected the commas and periods. In the early 1900s, it appears that the Fowler brothers (who wrote a famous British style guide called The King’s English) began lobbying to make the rules more about logic and less about the mechanics of typesetting. They won the British battle, but Americans didn’t adopt the change.

mingyli replied:
Sires, this is an **American** publication, produced by **Americans**. As such, we will conform to the native language, and accommodate our audience.

RahulTirumalareddy replied:
**An open letter:**
I agree that we are Americans. However, what is America but a melting-pot of cultures, where the best aspects of societies around the globe are pieced together so that any one who graces this land has an opportunity for success? Truly, the **American** thing to do here is not to blindly follow convention set by an older generation, but rather, to innovate and improve upon their wisdom. Much like the theory of the living Constitution, I believe that grammar is a set of standards that should constantly adapt to the changing circumstances and constraints that a people face, not the other way around. As such, I am in favor of the situational placement of commas/periods around quotation marks.

mingyli replied:
I offer you two pieces of evidence:

1. An [article](https://www.nytimes.com/2019/04/17/opinion/tsa-transgender.html) from The New York Times. You will note that all quotations contain punctuation inside.
2. An [article](https://www.theguardian.com/politics/2019/apr/17/theresa-may-could-put-off-queens-speech-amid-brexit-turmoil) from The Guardian, a British publication. You will note that all quotations contain punctuation inside.

RahulTirumalareddy replied:
I must clarify, though, that while I am a proponent of British grammar rules for their flexibility and utilitarian focus, spelling should be done in an American manner, as our audience is primarily American, and there is nothing intrinsic about the way a word is spelled, as opposed to how words are laid out grammatically.

mingyli replied:
If it is the goal of (dingers)[dingers-music.github.io] to enter the mainstream and entertain the general populace, we must apply proper written language rules.

RahulTirumalareddy replied:
> in order to “build the broadest possible consensus for our Brexit plans”.

> refusal to bring forward a Queen’s speech would be a “constitutional outrage”.

The two excerpts above were taken from The Guardian's article. To briefly summarize, when dialogue is molded to fit the context of a sentence, sometimes referred to as blending quotations, as in the examples above, the quoted portion is treated as an atomic unit, which explains why the punctuation is on the outside.

mingyli replied:
I concede

mingyli replied:
But I still bear a preference for American. You may create your posts with the disclaimer that your writing employs archaic British effects; I will continue to create my publications the *correct* way.

RahulTirumalareddy replied:
The polarization of today's grammar circles is quite disheartening. We must cast aside our grammatical preferences and realize that both parties just want what's best for our country semantically.

mingyli replied:
Allow us to subscribe to the [AP Style book](https://www.apstylebook.com/) convention to eliminate any debate

mluogh replied:
Sires, I believe us all to be logical men of the 21st century.

Let us cast aside all superstition and archaic remnants of the past!

mingyli reviewed (commented):
Great changes. I am impressed with your ability to rebase. Some minor changes

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+Here, as a woman attempts to engage in promiscuous behaviour, our hero denies
+her access to his room (and therefore himself). Furthermore, he corrects her --
+his desire for women cannot be compared to the vast ocean, but just a small
+pool, by comparison practically non-existent.
```
Turns out nonexistent is a single word

mluogh replied:
https://dictionary.cambridge.org/us/dictionary/english/non-existent

Cambridge dictionary allows both.

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+America (and more generally, the world at large) to accept what is a natural
+phenomenon is brave, commendable, and well crafted.
+
+For those further interested, my colleagues DJ Splash Drip and Kurosare have
```
it's kusorare

mluogh replied:
Good catch.

RahulTirumalareddy reviewed (changes requested):
Great fixes; just one little thing needs to be made and this can go live!

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+us/album/sicko-mode/1421241217?i=1421242781&app=music"
+height="150" frameborder="0"></iframe>
+
+The first beat of the song is mild and unassuming - a careful allegory chosen
```
em dash

mingyli commented:
Gentlesires, I refer you to the copy editor guidelines [here](https://www.iep.utm.edu/volunteer/copyedit/#5t)

RahulTirumalareddy commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+phenomenon is brave, commendable, and well crafted.
+
+For those further interested, my colleagues DJ Splash Drip and Kusorare have
+taken this analysis further in their song “[Dicko
```
Kind of reads awkwardly, as 'further' is repeated twice.

mingyli commented on _posts/2019-04-16-sicko-mode.md:
```diff
@@ -0,0 +1,100 @@
+for the women in his life, and another for the men. He must publicly proclaim
+“checks over stripes” (stripes on a multi coloured flag represent
+homosexuality). Despite his suffering, our strong and resilient protagonist
+still proclaims:
```
uses proclaims twice

RahulTirumalareddy reviewed (approved):
Thanks for sticking to it, Michael!
